docs/adr/0005-deploy-setup.md

```diff
@@ -45,11 +45,11 @@ Schema validation at Docker runtime should be taken out.
 
 ### Deployment
 
-A single `Jenkinsfile` in the [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository will tie the [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository and the separate 'packages' together. This will prevent having race conditions.
+~~A single `Jenkinsfile` in the [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository will tie the [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository and the separate 'packages' together. This will prevent having race conditions.~~
```

Comment:
and these strikethrough lines @bartjkdp

Reply:
Yes, please remove this 👍  also the repository Amsterdam/signalen can be deleted.

Reply:
heb die repo verwijderd @bartjkdp!